administration/intro.adoc

```diff
@@ -265,11 +266,47 @@ ____________________________________
 Note: Updates are not yet supported.
 ____________________________________
 
-When using the stable deployment flow, usually it is sufficient to re-apply the manifests for performing a
-rolling update:
+In both cases re-apply the manifests will not guarantee you the correct rolling update:
+
+[source,bash]
+----
+$ RELEASE=v0.15.0
+$ kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt-operator.yaml
+$ kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt-cr.yaml
+----
 
 [source,bash]
 ----
-$ RELEASE=v0.14.0
+$ RELEASE=v0.15.0
 $ kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt.yaml
 ----
+
+Delete
+~~~~~~
+
+For operator delployment you must first to delete `KV` custom resource and after delete the KubeVirt operator:
+
+[source,bash]
+----
+$ RELEASE=v0.15.0
+$ kubectl delete -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt-cr.yaml
+$ kubectl delete -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt-operator.yaml
+----
+
+______________________________________________________________________________________________________________________________________________________
+Note: If by mistake you deleted the operator first, the KV custom resource will stuck in the terminatig state,
```

Comment:
terminati*n*g (Also the state is upper case -> Terminating).

I am also not sure it makes sense to document this workaround for incorrect deletion steps in the main documentation but maybe that's just me.

Reply:
we already had such issues, so I prefer to leave it